llvm/lib/Target/AMDGPU/SIRegisterInfo.cpp

```diff
@@ -3614,3 +3614,14 @@ SIRegisterInfo::getSubRegAlignmentNumBits(const TargetRegisterClass *RC,
   }
   return 0;
 }
+
+SmallVector<std::string>
```

Comment:
probably should just be const char*, this will probably only ever be used with literals